website/docusaurus.config.js

```diff
@@ -0,0 +1,88 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: 'Spectacle',
+  tagline:
+    'A React.js based library for creating sleek presentations using JSX syntax that gives you the ability to live demo your code.',
+  url: 'https://formidable.com',
+  baseUrl: '/open-source/spectacle/',
+  onBrokenLinks: 'throw',
+  onBrokenMarkdownLinks: 'warn',
+  favicon: 'img/favicon.ico',
+  organizationName: 'formidablelabs',
+  projectName: 'spectacle',
+
+  presets: [
+    [
+      'classic',
+      /** @type {import('@docusaurus/preset-classic').Options} */
+      ({
+        docs: {
+          path: '../docs',
+          sidebarPath: require.resolve('./sidebars.js'),
+          editUrl:
+            'https://github.com/FormidableLabs/spectacle/tree/main/website',
```

Comment:
Is `editUrl` `docs` or `website`?

Reply:
We have this in docusaurus config:

```
editUrl: 'https://github.com/FormidableLabs/spectacle/tree/main/website',
path: '../docs',
```

With the addition of `'../docs'`, I have confirmed that these edit links do indeed point to the right place (which is the `docs` folder).